packages/components/src/components/sheet/sheet.tsx

```diff
@@ -165,7 +165,7 @@ export class Sheet extends LitElement {
   /**
    * When `position` is `"block-start"` or `"block-end"`, specifies the height of the component.
    *
-   * @deprecated Use the `height` property instead.
+   * @deprecated in v3.0, removal target v6.0 - Use the `height` property instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use the `height` property instead.
```